examples/client.rs

```diff
@@ -140,7 +140,22 @@ fn main() {
         Err(e) => panic!("initial send failed: {:?}", e),
     };
 
-    socket.send(&out[..write]).unwrap();
+    loop {
```

Comment:
Since we don't really care about the `Ok` value, this can probably be simplified to something like:
```rust
while let Err(e) = socket.send(&out[..write]) {
    if e.kind() == std::io::ErrorKind::WouldBlock {
        debug!("send() would block");
        break;
    }

    panic!("send() failed: {:?}", e);
}
```